src/components/sheet/sheet-action-buttons/SheetActionButtonRow.js

```diff
@@ -14,6 +14,18 @@ function renderButton(child) {
   return <FlexItem marginHorizontal={7.5}>{child}</FlexItem>;
 }
 
-export default function SheetActionButtonRow({ children }) {
+const ActionRowAndroid = styled.View`
+  flex-direction: row;
+  height: 44;
+  margin-vertical: 12;
+  margin-horizontal: 12;
```

Comment:
instead of defining both `margin-vertical` and `margin-horizontal` with the same value of `12` we should just define `margin: 12`